src/util/event-to-promise.test.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env jest */
```

Comment:
I am sorry could you elaborate? I didn't get what the functionality is expected out of the test.